src/libraries/System.Numerics.Tensors/ref/System.Numerics.Tensors.netcore.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace System.Buffers
 {
+    [System.Diagnostics.CodeAnalysis.Experimental("SNTEXP0001")]
```

Comment:
CC. @terrajobst, I don't think we have an existing format for these. So I've somewhat just followed what some other users (ASP.NET/Azure) are doing here.

Namely, `SNT` for `System.Numerics.Tensors`, then `EXP` for `Experimental`, then `0001` since this is the first ID.

Reply:
@terrajobst Should we instead use `SYSLIB2###` to follow suit of Obsoletions (`SYSLIB0###`) and Analyzers (`SYSLIB1###`)?

Reply:
I think we should stick to `SYSLIB` for all diagnostics (obsoletions, experimental, and analyzers).

My concern with having separate prefixes is that it will be very inconsistent across all the parties. It's hard enough to come up with a prefix that is short enough and identifies a single party that can keep track of the numbers to avoid duplicates. Asking folks to have separate prefixes for obsoletions, experimental, and general purposes analyzers *and* have that scheme make sense across all parties, is a bit too much IMHO. And frankly it doesn't seem to add much more value to the consumer as the way a given diagnostic needs to be addressed often differs between different cases within the same category anyway. So the expectation is: read the message and follow the link if you need more information.